google/cloud/storage/blob.py

```diff
@@ -3407,6 +3410,38 @@ def update_storage_class(
                 retry=retry,
             )
 
+    def open(
```

Comment:
Do you think it would be more readable to just do first class methods for each BlobReader/Writer combo? 

For example `def open_blob_reader()`

Reply:
It would be more readable, but this is (regrettably) the standard open() interface shared by all Python file-like I/O.

Reply:
thanks for clarifying.